src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/source/CoverageSourcePrinter.java

```diff
@@ -31,6 +31,11 @@ class CoverageSourcePrinter implements Serializable {
 
     private final int[] missedPerLine;
 
+    public static final String ALL_BRANCHES_COVERED = "All branches covered";
```

Comment:
We can use `Messages._All_Branches_Covered().toString(new Locale(""))` to get the default message from `Message.properties`, to avoiding keeping two same message in different places.